x/gov/types/params.go

```diff
@@ -0,0 +1,80 @@
+package types
+
+import (
+	"bytes"
+	"fmt"
+	sdk "github.com/pokt-network/posmint/types"
+	"strings"
+)
+
+// DefaultParamspace defines the default auth module parameter subspace
+const DefaultParamspace = ModuleName
+
+// Default parameter values
+const ()
+
+// Parameter keys
+var (
+	ACLKey      = []byte("acl")
+	DAOOwnerKey = []byte("daoOwner")
```

Comment:
Who is the DAO owner? Is this the name of the permission?

Reply:
DAOOwner is the name of the person who can send and receive funds from the bridge account